src/components/WwwFrame/PromotionalBanner/Banners/AppealBanner/AppealBanner.vue

```diff
@@ -348,4 +465,72 @@ export default {
 		// }
 	}
 }
+
+// GROW-230 changes
+.overlay-shown .sitewide-appeal-wrapper {
+	z-index: 2000;
```

Comment:
Seeing huge z-index's scares me a little. From what I gather you just want this to be higher than sitewide-overlay? Hopefully it can just be 1 z-index higher instead of 1000.

Reply:
Fix being committed now